src/components/Editor/EditorInput/EditorInput.test.js

```diff
@@ -109,3 +120,104 @@ describe("Tab interactions", () => {
     );
   });
 });
+
+describe("On mobile", () => {
+  let store;
+  beforeEach(() => {
+    setMedia({
+      width: "400px",
+    });
+
+    const middlewares = [];
+    const mockStore = configureStore(middlewares);
+    const initialState = {
+      editor: {
+        project: {
+          components: [
+            {
+              name: "main",
+              extension: "py",
+              content: 'print("hello")',
+            },
+            {
+              name: "a",
+              extension: "py",
+              content: "# Your code here",
+            },
+          ],
+        },
+        openFiles: [["main.py", "a.py"]],
+        focussedFileIndices: [1],
+      },
+      auth: {
+        user: null,
+      },
+    };
+    store = mockStore(initialState);
+    render(
+      <Provider store={store}>
+        <div id="app">
+          <EditorInput />
+        </div>
+      </Provider>,
+    );
+  });
+
+  test("Run button to be in the tab bar", () => {
+    screen.debug();
```

Comment:
this one might be go away as well

Reply:
Ah yes, this is also something I missed 👍